pkgs/development/tools/misc/saleae-logic/default.nix

```diff
@@ -91,7 +92,7 @@ stdenv.mkDerivation rec {
     description = "Software for Saleae logic analyzers";
     homepage = http://www.saleae.com/;
     license = licenses.unfree;
-    platforms = [ "x86_64-linux" "i686-linux" ];
+    platforms = subtractLists platforms.i686 platforms.linux;
```

Comment:
I hadn't seen this before, so I evaluated it in nix repl. Turns out that platforms.i686 is of a different type than platforms.linux, so nothing gets subtracted:

```
$ nix repl '<nixpkgs>'
nix-repl> (lib.subtractLists lib.platforms.i686 lib.platforms.linux) == lib.platforms.linux
true
```

Actually, I'm confused about platforms.linux no longer being a simple list of strings (["x86_64-linux" ...]). (I'm not that active in nixpkgs development.)

I guess I'd just do ```platforms = [ "x86_64-linux" ]``` for now, unless someone can tell us what the proper way to limit to 64-bit linux with platforms.* attribute.

Reply:
With the "x86_64-linux" assert at the top of the expression, I guess using "platforms = platforms.linux" is fine as well.

Reply:
Done